### PR TITLE
Add jcli support for EVM mapping certificates

### PR DESCRIPTION
**Pull request by saibatizoku**
Need to add handling of the new Certificate/Fragment EvmMapping in jormungandr, and also update jcli to generate and sign this new type of Certificate/Fragment.

### CODE REVIEW

**Comment by Mr-Leshiy** [comment]
need to update with this update https://github.com/input-output-hk/chain-libs/pull/774#pullrequestreview-911328609

File: jormungandr-lib/src/interfaces/certificate.rs [hunk 1]
```diff
@@ -217,6 +224,11 @@ impl property::Serialize for SignedCertificate {
                 writer.write_all(c.serialize().as_slice())?;
                 writer.write_all(a.serialize_in(ByteBuilder::new()).finalize().as_slice())?;
             }
+            certificate::SignedCertificate::EvmMapping(c, a) => {
+                writer.write_all(&[15])?;
```
Comment (Zeegomo) [inline jormungandr-lib/src/interfaces/certificate.rs:228]: why 15?

**Review by Zeegomo — COMMENTED** [review]
I've also pushed some changes, but they are not appearing for some GitHub quirk

File: jormungandr-lib/src/interfaces/certificate.rs (hunk 1 above)
Comment (saibatizoku) [reply jormungandr-lib/src/interfaces/certificate.rs:228]: From what I can tell, the serialization of these certificates use the same numbering scheme as the tag enum for fragments in chain-libs

https://github.com/input-output-hk/chain-libs/blob/master/chain-impl-mockchain/src/fragment/mod.rs#L53-L70

Comment (Zeegomo) [reply jormungandr-lib/src/interfaces/certificate.rs:228]: Do they? most of `Certificate` and `SignedCertificate` overlap anyway and those are at two different levels

**Review by Zeegomo — APPROVED** [review]

**Pull request merged**